### PR TITLE
Adding stories.yaml and requirements.yaml to repo

### PR DESCRIPTION
**Pull request by seth127**
Moving validation stories and requirements into YAML files in the repo, as opposed to external Googlesheets.

Below is some sample R code to generate validation docs from these YAML files (and test results that are dumped to S3 from our drone CI). The intention is to add something like this as another step in the CI that will generate the docs and dump them to S3 anytime a tag is pushed.

```r
library(mrgvalprep)
library(mrgvalidate)

# set working dir to the validation/ dir in the BBI repo, then
# copy the results.json and summary.json from a drone run into this directory
test_dir <- "test_output"

# parse raw Go test results and write to csv
test_res <- parse_golang_test_json(file.path(test_dir, "results.json"))
readr::write_csv(test_res, file.path(test_dir, "summary.csv"))

# read in spec YAML files
spec <- read_spec_yaml(
  "stories.yaml",
  "requirements.yaml"
)

create_validation_docs(
  "bbi",
  "3.1.1.9000",
  spec,
  auto_test_dir = test_dir,
)
```

### CODE REVIEW

File: validation/requirements.yaml [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+POSTEX-R002:
+  description: Post execution succeeds
+  tests:
+  - INT-POSTEX-002
+SGE-R001:
+  description: BbiCompletesSGEExecution
+  tests:
+  - INT-SGE-001
+SGE-R002:
+  description: BBBI completes parallel SGE execution
```
Comment (kyleam) [inline validation/requirements.yaml:98]: s/BBBI/BBI/

File: validation/requirements.yaml [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+POSTEX-R001:
+  description: KVP expansion works
+  tests:
+  - INT-POSTEX-001
+POSTEX-R002:
+  description: Post execution succeeds
+  tests:
+  - INT-POSTEX-002
+SGE-R001:
+  description: BbiCompletesSGEExecution
```
Comment (kyleam) [inline validation/requirements.yaml:94]: "BBI completes SGE execution"?

**Review by kyleam — CHANGES_REQUESTED** [review]
Looks fine to me aside from a typo and an an unexpanded description.

File: validation/requirements.yaml (hunk 1 above)
Comment (seth127) [reply validation/requirements.yaml:98]: When you run it in parallel it gets an extra B.

**Review by kyleam — APPROVED** [review]

**Pull request merged**